### PR TITLE
Correcting links.

### PR DESCRIPTION
**Pull request by grainnejenningsRH**
Corrects links in rstudio disclaimers downstream.

## Merge criteria:

- [ ] The commits are squashed in a cohesive manner and have meaningful messages.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

File: modules/configuring-your-ide.adoc [hunk 1]
```diff
@@ -70,7 +70,7 @@ ifdef::cloud-service[]
 [IMPORTANT] 
 ====
 *Disclaimer:* 
-{org-name} supports managing workbenches in {productname-short}. However, {org-name} does not provide support for the RStudio software. RStudio Server is available through link:rstudio.org[rstudio.org] and is subject to their licensing terms. You should review their licensing terms before you use this sample workbench.
+{org-name} supports managing workbenches in {productname-short}. However, {org-name} does not provide support for the RStudio software. RStudio Server is available through link:rstudio.org[https://rstudio.org/] and is subject to their licensing terms. You should review their licensing terms before you use this sample workbench.
```
Comment (chtyler) [inline modules/configuring-your-ide.adoc:73]: I checked the link and it's fine. However, would this be a good time to remove the word "should"?

File: modules/options-for-notebook-server-environments.adoc [hunk 2]
```diff
@@ -351,7 +351,7 @@ a| * Python 3.9
 [IMPORTANT] 
 ====
 *Disclaimer:* +
-{org-name} supports managing workbenches in {productname-short}. However, {org-name} does not provide support for the RStudio software. RStudio Server is available through link:rstudio.org[rstudio.org] and is subject to their licensing terms, which should be reviewed prior to use of this sample workbench. +
+{org-name} supports managing workbenches in {productname-short}. However, {org-name} does not provide support for the RStudio software. RStudio Server is available through link:rstudio.org[https://rstudio.org/] and is subject to their licensing terms, which should be reviewed prior to use of this sample workbench. +
```
Comment (chtyler) [inline modules/options-for-notebook-server-environments.adoc:354]: Same as first comment.

**Review by chtyler — APPROVED** [review]

File: modules/configuring-your-ide.adoc (hunk 1 above)
Comment (grainnejenningsRH) [reply modules/configuring-your-ide.adoc]: Agreed - updated to remove.

File: modules/options-for-notebook-server-environments.adoc (hunk 2 above)
Comment (grainnejenningsRH) [reply modules/options-for-notebook-server-environments.adoc]: Updated to remove - also applied to all other instances of 'should' in the disclaimers.

**Review by chtyler — APPROVED** [review]

**Pull request merged**